content/open-source/melange/faq.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "melange FAQs"
+type: "article"
+lead: "Frequently asked questions about melange"
+description: "Frequently asked questions about melange"
+date: 2022-10-17T11:07:52+02:00
+lastmod: 2022-10-17T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "melange"
+weight: 900
+toc: true
+---
+
+### How are melange packages defined?
+melange apks are defined declaratively using a YAML file.
+
+### Are melange apks compatible with Alpine?
+Yes, melange apks are built to be compatible with apk-based systems including Alpine.
+
+### Can I mix Alpine and Wolfi package repositories to create my melange build environment?
+No, it's not possible to mix Alpine apks with Wolfi apks. If you have unmet dependencies, you'll need to build those first as separate packages.
+
+### Is it mandatory to sign packages with a melange key?
```

Comment:
There is a weird space on the deploy preview, just fyi. Let's monitor the real deploy

<img width="736" alt="Screen Shot 2022-11-09 at 12 59 02 PM" src="https://user-images.githubusercontent.com/5977693/200905334-87b2712d-8d85-4a71-86cd-252844ad8cba.png">

Reply:
Hm that's strange! I don't see it here. Maybe it's some font-related thing, let's see how it looks like when we merge.